modules/azure-control-plane-sizing.adoc

```diff
@@ -6,5 +6,5 @@
 [id="azure-control-plane-sizing_{context}"]
 = Recommended Control Plane Sizing for Azure
 
-For Azure, It is recommended to use Control Plane nodes with "Standard_D16s_v3" instances for improved performance and stability, as using "Standard_D8s_v3" instances led to firing critical alerts such as etcdGRPCRequestsSlow. 
+For Azure, It is recommended to use Control Plane nodes with at least "Standard_D16s_v3" instance type for improved performance and stability, as using "Standard_D8s_v3" instances led to firing critical alerts such as etcdGRPCRequestsSlow. 
```

Comment:
We might want to document the OpenShift version and environment test details @shahsahil264. Thoughts?